internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/admin/InstrumentedPubSubAdminAdapter.java

```diff
@@ -0,0 +1,165 @@
+package com.linkedin.venice.pubsub.adapter.kafka.admin;
```

Comment:
Minor: this is might be previously left problem. I think we may need to move this to higher module kafka, since other pub sub adpaters could also use this.

Reply:
yeah that's right. Moved it to pubsub api module. Thanks!